owlplug-client/src/main/java/com/owlplug/core/controllers/OptionsController.java

```diff
@@ -144,7 +153,16 @@ public void initialize() {
 
     storeDirectoryCheckBox.selectedProperty().addListener((observable, oldValue, newValue) -> {
       this.getPreferences().putBoolean(ApplicationDefaults.STORE_DIRECTORY_ENABLED_KEY, newValue);
+      double width = newValue ? 150 : 0;
```

Comment:
the UI here is a bit different, I actually hide the path segments based on the toggles, so the path 'looks' like what it will end up like.

Reply:
LGTM